.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: node_js
+dist: trusty
+sudo: required
+node_js: 6
+env:
+  - PATH=$HOME/purescript:$PATH
+install:
+  - TAG=$(wget -q -O - https://github.com/purescript/purescript/releases/latest --server-response --max-redirect 0 2>&1 | sed -n -e 's/.*Location:.*tag\///p')
```

Comment:
Is it possible to manually specify the versions of purescript you want builds for?